### PR TITLE
[AJUN-495] Runtime Upgrade for Async Backing

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

Updates runtime to enabled 6s blocks through async-backing

## Type of changes

- [ ] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [x] `feat`: Changes to add a new feature
- [ ] `fix`: Changes to fix a bug
- [ ] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [ ] Tests for the changes have been added
- [ ] Necessary documentation is added (if appropriate)
- [ ] Formatted with `cargo fmt --all`
- [ ] Linted with `cargo clippy --all-features --all-targets`
- [ ] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

File: runtime/bajun/src/lib.rs [hunk 1]
```diff
@@ -446,7 +446,7 @@ impl pallet_timestamp::Config for Runtime {
 	/// A timestamp: milliseconds since the unix epoch.
 	type Moment = u64;
 	type OnTimestampSet = Aura;
-	type MinimumPeriod = ConstU64<0>;
+	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
```
Comment (clangenb) [inline runtime/bajun/src/lib.rs]: This change is wrong IMO, it should stay at 0 if I read this correctly: https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-3---activate-async-backing

(But maybe it was wrong that it was 0 before XD)

**Review by clangenb — COMMENTED** [review]
Nice, except for one comment I think this looks good. 

I reviewed that:
* The changes correspond to the guide https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-3---activate-async-backing
* Double checked that the service aligns with the parachain `polkadot-sdk\cumulus\polkadot-parachain\src\service.rs`
* Checked that the runtime matches the `polkadot-sdk\templates\parachain\runtime\src` although the runtime is so minimal that not everything is covered.

File: runtime/bajun/src/lib.rs (hunk 1 above)
Comment (DidacSF) [reply runtime/bajun/src/lib.rs]: True, you are right about that!

**Review by clangenb — APPROVED** [review]
Wonderful. Looks good to me now!

**Pull request merged**